BLL/SearchDrinkByName.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace BLL
+{
+    public static class SearchDrinkByName
+    {
+        public static List<Drink> SearchByName(string textToSearch, List<Drink> drinksListToSearch)
+        {
+            var drinksFound = new List<Drink>();
+
+            foreach (var drink in drinksListToSearch)
+            {
+                if (drink.Name.ToLower().Contains(textToSearch.ToLower()))
```

Comment:
That way is correct! But if you want to go even further, you can do the same using some additional argument in contains method:
`drink.Name.Contains(textToSearch, StringComparison.InvariantCultureIgnoreCase)` 

I'll also tell you a little spoiler, that soon you will be able to write this entire method in +- single line ;) If you remind me about that in our Project workshops I'll show you live how :)